protected/modules/OphDrPrescription/views/report/prescribedDrugs.php

```diff
@@ -92,7 +92,7 @@
                         </div>
                     </div>
                     <div class="large-1 column end">
-                        <img class="autocomplete-loader" style="display: none;" src="/assets/9504fa83/img/ajax-loader.gif" alt="loading...">
+                      <img class="autocomplete-loader" style="display: none;" src="<?php echo Yii::app()->assetManager->createUrl('img/ajax-loader.gif')?>" alt="loading..." />&nbsp;
```

Comment:
Is there a reason for the added whitespace character at the end?

Reply:
I think it was added by mistake.